cdap-docs/application-templates/source/etl/templates/transformations/projection.rst

```diff
@@ -6,19 +6,56 @@
 Transformations: Projection
 ===========================
 
-.. rubric:: Description: Projection Transform: Drop, Rename, and Cast Fields
+.. rubric:: Description
 
-Projection transform that lets you drop, rename, and cast fields to a different type.
+A transform that can drop fields, rename fields, and cast fields to a different type.
```

Comment:
A transform that can drop fields, cast fields to a different type, and rename fields.
[let's keep them in order.]